doc/conf.py

```diff
@@ -0,0 +1,13 @@
+"""Sphinx configuration file for an LSST stack package.
+
+This configuration only affects single-package Sphinx documenation builds.
+"""
+
+from documenteer.conf.pipelinespkg import *  # noqa: F403, import *
+
+project = "felis"
+html_theme_options["logotext"] = project  # noqa: F405, unknown name
+html_title = project
+html_short_title = project
+doxylink = {}
+exclude_patterns = ["changes/*"]
```

Comment:
Maybe I should encourage the new data engineer to start using towncrier when they start developing this package.